### PR TITLE
Fixed search box

### PR DESCRIPTION
**Pull request by timschoch**
This fixes the search box to the top of the browser window. By this you can easely seach on if the first element was not the one you where looking for.
First Commit is to fix the search bar, second one prevents the item titles from hiding behind the search bar

### CODE REVIEW

**Comment by HugoGiraudel** [comment]
Could you please:

* head your PR to `develop`, not `master`
* synchronise your branch with `develop` so you don't include my last commit

Cheers. :)

File: scss/partials/_header.scss [hunk 1]
```diff
@@ -3,11 +3,20 @@
  */
 .header {
   @include clearfix;
+  @include fixed( $left: $sidebar-width, $top: 0, $right: 0 );
   box-shadow: 0 2px 5px rgba(0,0,0,.26);
   padding: 1em 0;
   background: $primary-color;
   color: $light-grey;
-  position: relative;
+  z-index: z("dropdown");
```
Comment (HugoGiraudel) [inline scss/partials/_header.scss]: Single quotes. :)

**Comment by timschoch** [comment]
Merde. No idea how I could change master do develop, so I created a new change request. hope this time it's right :)

**Comment by HugoGiraudel** [comment]
> No idea how I could change master do develop, so I created a new change request. hope this time it's right. :)

You can't, so you did the right thing. :)

**Pull request closed without merging**